_posts/cleaning-up-your-test-data-builders.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "Cleaning Up Your Test Data Builders"
+draft: true
+tags: [ "Cleaner Code", "Test Automation" ]
+---
+
+I've been using the [Test Data Builder](http://www.natpryce.com/articles/000714.html) pattern for quite a while now. It's really improved the readability of my test automation code. I've always had the nagging feeling though that it could be better.
+
+Inspired by a talk I attended a couple of months ago I decided to refactor some of my test data builders. In this post I'll go through some of the steps I've taken.
+
+The code example below is my initial take on a builder to create a simple Person object. It has a name, gender and address.
```

Comment:
Maybe you could give this code example in your first section and explain why this example gave you a nagging feeling. After that you can introduce what inspired you to change it.